test/e2e/autoscale_test.go

```diff
@@ -62,22 +62,24 @@ func TestRPSBasedAutoscaleUpCountPods(t *testing.T) {
 func runAutoscaleUpCountPods(t *testing.T, class, metric string) {
 	target := containerConcurrency
 	if metric == autoscaling.RPS {
-		target = 10
+		target = rpsTarget
 	}
 
 	ctx := SetupSvc(t, class, metric, target, targetUtilization)
 
 	ctx.t.Log("The autoscaler spins up additional replicas when traffic increases.")
-	// note: without the warm-up / gradual increase of load the test is retrieving a 503 (overload) from the envoy
+	// Note: without the warm-up / gradual increase of load the test is
+	// receiving 503 responses (overload) from the envoy.
```

Comment:
🤔  I'll check the issue, but 10 RPS serving 503s is concerning.

Reply:
See my comment on the issue (I think we need to dig into why we're seeing delays, maybe highlighted by chaos?).  Why aren't 503s outright failures here regardless? 🤔

Reply:
This comment is old. I am not even sure it is even relevant anymore.
I think the delays are due to the kube determining readiness, kubeproxy/endpoints informer propagating the information.